Polkadot spec wiki/guide integration (#4009)
w3f/polkadot-wiki: docs/build/build-substrate.md

### Substrate (full node) vs. Substrate connect (light client)

> A light client is a client that lets you utilize all the possibilities of the chain, but it does not

Reviewer: ```suggestion
A light client lets you utilize all basic features of the chain such as fetching data and transferring tokens, but it does not
```

> require you to run a full copy of the entire blockchain. Light clients fetch the required data that

Reviewer: ```suggestion
require you to run a full copy of the entire blockchain or having to trust remote peers. Light clients fetch the required data that
```

they need from a {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} node with an
associated proof to validate the data. This makes it possible to interact with the
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} network without requiring to run a

> full node or having to trust the remote peers.

Reviewer: ```suggestion
associated proof to validate the data.
```

Reviewer: I removed this bit as the information is already present at the beginning of the paragraph
